libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -175,6 +177,22 @@ const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     AP_GROUPINFO("2_YAW_CORR", 18, AP_Proximity, _yaw_correction[1], 0),
 #endif
 
+    // @Param: _OPTIONS
```

Comment:
as discussed let's remove these extra parameters.  It's just getting too confusing for users to figure out how to set things up.  Once @meee1 has completed MP's obstacle distance viewer (and we've confirmed it works) then we should stop sending the distance-sensor messages from the proximity library.

Reply:
Done.